scipy/spatial/transform/rotation.pyx

```diff
@@ -1572,6 +1573,30 @@ cdef class Rotation:
         else:
             return np.asarray(mrps)
 
+    @classmethod
+    def concatenate(cls, rotations):
+        """Concatenate a sequence of `Rotation` objects.
+
+        Parameters
+        ----------
+        rotations : sequence of `Rotation` objects
+            The rotations to concatenate.
+
+        Returns
+        -------
+        concatenated : `Rotation` instance
+            The concatenated rotations.
+
+        Notes
+        -----
+        .. versionadded:: 1.8.0
+        """
+        if not all([isinstance(x, Rotation) for x in rotations]):
```

Comment:
I assume that `[]` won't be necessary and it even more idiomatic to omit it here.

Reply:
Thanks Nikolay. Done.